docs/source/guides/staking_guide.rst

```diff
@@ -392,30 +392,31 @@ since the contract can't perform transactions, and it's the preallocation user
 (also known as the "`beneficiary`" of the contract)
 who has to perform staking operations.
 
+As part of the preallocation process, beneficiaries receive an allocation file,
+containing the ETH addresses of their beneficiary account and corresponding
+preallocation contract.
+
 In general, preallocation users can use all staking-related operations offered
-by the CLI in the same way as described above, except that they have to include
-the ``--escrow`` flag to indicate they are acting as beneficiaries.
+by the CLI in the same way as described above, except that they have to specify
+the path to the allocation file using the option ``--allocation-filepath PATH``.
 
 For example, to create a stake:
 
 .. code:: bash
 
-    (nucypher)$ nucypher stake create --hw-wallet --escrow
+    (nucypher)$ nucypher stake create --hw-wallet --allocation-filepath PATH
 
 
 Or to set a worker:
 
 .. code:: bash
 
-    (nucypher)$ nucypher stake set-worker --hw-wallet --escrow
-
+    (nucypher)$ nucypher stake set-worker --hw-wallet --allocation-filepath PATH
 
-As an alternative to the ``--escrow`` flag, preallocation users can directly specify
-their beneficiary address with the ``--beneficiary-address ADDRESS`` flag.
 
-The CLI will automatically look for a file named ``allocation_registry.json`` in
-the default installation path, although a custom path can be used with the flag
-``--allocation-filepath /path/to/allocation_registry.json``.
+As an alternative to the ``--allocation-filepath`` flag, preallocation users
+can directly specify their beneficiary and staking contract addresses with the
+``--beneficiary-address ADDRESS`` and ``--staking-address ADDRESS``, respectively.
```

Comment:
So, staking address is address of the contract which will stake for this beneficiary?

Reply:
Yes, exactly.